querier.hh

```diff
@@ -232,13 +232,24 @@ public:
     template <typename Consumer>
     requires CompactedFragmentsConsumer<Consumer>
     auto consume_page(Consumer&& consumer,
+            tracing::trace_state_ptr trace_ptr,
```

Comment:
Why not add as a last param with a default to nullptr to cause less churn?

Reply:
Can do.